proto/regen/ecocredit/basket/v1/events.proto

```diff
@@ -34,8 +34,8 @@ message EventPut {
   string amount = 4;
 }
 
-// EventTake is an event emitted when credits are taken from a basket starting
-// from the oldest credits first.
+// DEPRECATED: EventTake is an event emitted when credits are taken from a
+// basket starting from the oldest credits first.
```

Comment:
This event is not deprecated and only the `credits` field should be marked as deprecated.

```suggestion
// EventTake is an event emitted when credits are taken from a
// basket starting from the oldest credits first.
```

Reply:
I'm unable to make a suggestion for the `credits` field, which is the field we want to deprecate, but I think we should follow the same format as the sdk.

```
  // credits are the credits that were taken from the basket.
  // Deprecated: this field has been deprecated and will always return empty.
  repeated BasketCredit credits = 3;
```

Reply:
Same as above. We can also say `this field will be removed in the next version` and return the credits in the event. I think either way would be ok but we may want to provide backwards compatibility. What do you think?